frontends/web/src/api/backend.ts

```diff
@@ -125,3 +125,15 @@ export const subscribeAuth = (
 ) => (
   subscribeEndpoint('auth', cb)
 );
+
+/**
+ * Returns a function that subscribes a callback on "keystores". It is called whenever a keystore
+ * is connected or disconnected.
+ */
+export const syncKeystores = (
+  cb: () => void
```

Comment:
I thought keystores returns TKeystores.

see this PR here: https://github.com/digitalbitbox/bitbox-wallet-app/pull/2307/files#diff-8bd5c5b00b0d3f20c4f935b36bf42765f3a2ccbccdd5f982e398f284127fb02bR26

Reply:
This PR will add typed api/keystores  https://github.com/digitalbitbox/bitbox-wallet-app/pull/2389

Reply:
rebased, thanks!